File: Code Python/Exercicio-6/repeticao-for.py
#aula 24-09-2019
print('='*50)
print(" " * 19, "Atividade")
print('='*50)
nalunos = float(input("Digite o numero de alunos da turma: "))
print('='*50)
cont = 1
total = 0
while cont <= nalunos:
    nota = float(input(f"@ Digite a nota {cont}: "))
    total = float(total + nota)
    cont = cont + 1
print('='*50)
media = total/nalunos
print(' ' * 11, f"A media da turma eh: {media}")
print('='*50)

File: Code_C/Exercicio-06/repeticao-for.c
#include <stdio.h>
#include <stdlib.h>

void pausar(){
	printf("\nPressione alguma tecla para continuar...");
	getch();
}

int main(){
	
	int nalunos, cont=1;
	float nota, total, media;
	
	printf("Digite o numero de alunos da turma: ");
	scanf("%d", &nalunos);
	
	while(cont <= nalunos){
		printf("Digite a nota %d: ", cont);
		scanf("%f", &nota);
		total = total + nota;
		cont++;		
	}
	
	media = total/nalunos;
	
	printf("A media da turma eh: %f \n", media);
	
	pausar();
	return(0);
}
